### PR TITLE
chore: logging 4 – remove prints from config directory

### PR DESCRIPTION
**Pull request by hollandjg**
builds on
- #177

### CODE REVIEW

File: src/icesat2_tracks/config/generate_colors.py [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+
 mconfig['paths']
 path=mconfig['paths']['config']
 A = M_color.ase_to_json(path+'color_def.ase')
 
 B=dict()
 for i in A[0]['swatches']:
     B[i['name']] = i['data']['values']
-    print(i['name'] + '  ' + str(i['data']['values']))
+    _logger.debug("%s %s", i['name'], str(i['data']['values']))
```
Comment (cpaniaguam) [inline src/icesat2_tracks/config/generate_colors.py:15]:
```suggestion
    _logger.debug("swatch name: %s, values: %s", i['name'], i['data']['values'])
```

**Review by cpaniaguam — CHANGES_REQUESTED** [review]
This one was easy, thanks! Just one change here.

**Review by cpaniaguam — DISMISSED** [review]
👍

**Review by kmilo9999 — APPROVED** [review]
Looks good

**Pull request merged**